activities/PHQ-9/phq9_schema.jsonld

```diff
@@ -1,27 +1,84 @@
 {
+    "@context": "https://raw.githubusercontent.com/sanuann/schema-standardization/master/activities/PHQ-9/context.jsonld",
+    "@id": "/templates/phq9-template",
     "@type": "https://schema.metadatacenter.org/core/Template",
-    "@context": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/context.jsonld",
-    "@id": "#phq9-schema",
-    "label": "Patient Health Questionnaire-9 ",
-    "schema:description": "PHQ-9 Assessment Questionnaire",
+    "title": "PHQ-9 form schema",
+    "schema:description": "PHQ-9 assessment schema",
     "schema:schemaVersion": "0.0.1",
-    "question": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/question.jsonld",
-    "phq9_item:phq9_1": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_1.jsonld",
-    "phq9_item:phq9_2": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_2.jsonld",
-    "phq9_item:phq9_3": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_3.jsonld",
-    "phq9_item:phq9_4": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_4.jsonld",
-    "phq9_item:phq9_5": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_5.jsonld",
-    "phq9_item:phq9_6": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_6.jsonld",
-    "phq9_item:phq9_7": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_7.jsonld",
-    "phq9_item:phq9_8": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_8.jsonld",
-    "phq9_item:phq9_9": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_9.jsonld",
-    "phq9_item:phq9_10": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_10.jsonld",
-    "phq9_item:phq9_total_score": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_total_score.jsonld",
-    "phq9_item:phq9_category": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/PHQ-9/items/phq9_category.jsonld",
+    "preamble": [{
+        "@language": "en",
+        "@value": "Over the last 2 weeks, how often have you been bothered by any of the following problems? (Use “✔” to indicate your answer) ENGLISH"
+    },
+        {
+            "@language": "es",
+            "@value": "Over the last 2 weeks, how often have you been bothered by any of the following problems? (Use “✔” to indicate your answer) SPANISH"
+
+        }
+    ],
+    "phq9_1": {
```

Comment:
each of these keys should be defined in @context. see example here:

https://w3c.github.io/json-ld-syntax/#ex-20-combining-external-and-local-contexts

Reply:
I have refactored the contexts. Please review.